evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -112,29 +116,20 @@ def weight_info(evaluation):
 def statename(state):
     return STATE_NAMES.get(state)
 
-
 @register.filter
 def statedescription(state):
     return STATE_DESCRIPTIONS.get(state)
 
-
 @register.filter
 def approval_state_values(state):
     if state in APPROVAL_STATES:
         return APPROVAL_STATES[state]
-    if state in ['in_evaluation', 'evaluated', 'reviewed', 'published']:
-        return APPROVAL_STATES['approved']
-    return None
+    return APPROVAL_STATES[Evaluation.State.APPROVED]
```

Comment:
I verified all the callers. Should we still add a check, in case a new user of this filter ever messes it up?

Reply:
Hmm so this could break if someone added a new state that should not be approved. However, I think when adding a new state, one would look at all occurrences of APPROVAL_STATES anyway, so I'm fine with how it is right now.